charts/kafka-queues/Chart.yaml

```
@@ -0,0 +1,20 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: v2
+appVersion: "1.0"
+description: A Helm chart to create the Kafka queues
+name: kafka-queues
```

can we make this name more descriptive and therefore also the name of the charts directory
```suggestion
name: create-palisade-kafka-queues
```